editor/meta-boxes/index.js

```diff
@@ -6,7 +6,7 @@ import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
-import MetaBoxesIframe from './meta-boxes-iframe';
+import MetaBoxesArea from './meta-boxes-area';
 import MetaBoxesPanel from './meta-boxes-panel';
```

Comment:
`meta-boxes-panel/style.scss` still contains a text reference to "loaded from within the iframe"